roachtest: activerecord failed

[(roachtest).activerecord failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2226493&tab=buildLog) on [master@fb25dbe53c2bc99e673c292b71d73d30247a7450](https://github.com/cockroachdb/cockroach/commits/fb25dbe53c2bc99e673c292b71d73d30247a7450):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/activerecord/run_1
	orm_helpers.go:218,orm_helpers.go:144,activerecord.go:222,activerecord.go:234,test_runner.go:754: 
		Tests run on Cockroach v20.2.0-alpha.1-2818-gfb25dbe53c
		Tests run against activerecord 5.2.4.3
		5429 Total Tests Run
		5428 tests passed
		1 test failed
		17 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: FixturesTest#test_create_fixtures (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList20_2) is available in the artifacts' activerecord log

```

<details><summary>More</summary><p>


Artifacts: [/activerecord](https://teamcity.cockroachdb.com/viewLog.html?buildId=2226493&tab=artifacts#/activerecord)
Related:
- #53512 roachtest: activerecord failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-provisional_202008261913_v20.2.0-beta.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-provisional_202008261913_v20.2.0-beta.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Aactiverecord.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/activerecord_blocklist.go
@@ -30,7 +30,9 @@ var activeRecordBlockList20_2 = blocklist{}
 
 var activeRecordBlockList20_1 = blocklist{}
 
-var activeRecordIgnoreList20_2 = blocklist{}
+var activeRecordIgnoreList20_2 = blocklist{
+	"FixturesTest#test_create_fixtures": "flaky - FK constraint violated sometimes when loading all fixture data",
+}
 
 var activeRecordIgnoreList20_1 = blocklist{
 	"FixturesTest#test_create_fixtures": "flaky - FK constraint violated sometimes when loading all fixture data",

